components/clinical/timeline/src/atoms/timeline-time.tsx

```diff
@@ -7,9 +7,33 @@ import {
   TimelineDomainResourceType,
 } from '@ltht-react/types'
 import { formatTime } from '@ltht-react/utils'
+import Icon from "@ltht-react/icon";
+import styled from "@emotion/styled";
 
-const TimelineTime: FC<Props> = ({ domainResource, domainResourceType, ...rest }) => {
+const StyledLeftTimeIcon = styled(Icon)`
+  cursor: pointer;
+  display: inline-block;
+  padding-right: 0.25rem;
```

Comment:
Can we have some :hover css styling so it makes it even more obvious its interactable?